meshio/med_io.py

```diff
@@ -46,11 +47,29 @@ def read(filename):
         assert len(ts) == 1, "Must only contain exactly 1 time-step"
         mesh = mesh[list(ts)[0]]
 
+    # Read nodal and cell data if they exist
+    try:
+        cha = f["CHA"]  # champs (fields) in french
```

Comment:
I can't get over the French naming scheme. :laughing:

Reply:
That's really silly!